lib/mixins/fixable.rb

```diff
@@ -0,0 +1,10 @@
+module Fixable
+
+  def fix_price(row_hash)
```

Comment:
I may remove this module altogether (since I don't think this will be used anywhere besides the rake task) and limit price fixing to `Item` and `InvoiceItem` since those are the only resources that need conversions for prices.

Reply:
This is something that could go in the helper class I described in my above comment